packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableDetails.tsx

```diff
@@ -0,0 +1,241 @@
+                    <div className="flex space-x-2">
+                      <NetworkImage chainId={destinationChainId} />
+                      <span>{destinationNetworkName}</span>
+                    </div>
+                  </div>
+                </DetailsBox>
+
+                <DetailsBox header="Bridge">
+                  <div className="flex space-x-2">
+                    <Image
+                      alt="Bridge logo"
+                      src={tx.isCctp ? CctpLogoColor : ArbitrumLogo}
+                      width={16}
+                      height={16}
+                    />
+
+                    {tx.isCctp ? (
+                      <span>
+                        CCTP{' '}
+                        <span className="text-white/70">
+                          (Cross-Chain Transfer Protocol)
+                        </span>
+                      </span>
+                    ) : (
+                      <span>Arbitrum&apos;s native bridge</span>
+                    )}
+                  </div>
+                </DetailsBox>
+
+                {(isDifferentSourceAddress ||
+                  isDifferentDestinationAddress) && (
+                  <DetailsBox header="Custom Address">
+                    {isDifferentSourceAddress && (
+                      <span className="text-xs">
+                        Funds received from{' '}
+                        <ExternalLink
+                          className="arb-hover underline"
+                          href={`${getExplorerUrl(sourceChainId)}/address/${
+                            tx.sender
+                          }`}
+                        >
+                          {shortenAddress(String(tx.sender))}
+                        </ExternalLink>
+                      </span>
+                    )}
+                    {isDifferentDestinationAddress && (
+                      <span className="text-xs">
+                        Funds sent to{' '}
+                        <ExternalLink
+                          className="arb-hover underline"
+                          href={`${getExplorerUrl(
+                            destinationChainId
+                          )}/address/${tx.destination}`}
+                        >
+                          {shortenAddress(String(tx.destination))}
+                        </ExternalLink>
+                      </span>
+                    )}
+                  </DetailsBox>
+                )}
+
+                <DetailsBox>
+                  <TransactionsTableDetailsSteps tx={tx} address={address} />
+                </DetailsBox>
+
+                <div className="flex justify-end">
+                  <ExternalLink href={GET_HELP_LINK}>
+                    <Button
+                      variant="secondary"
+                      className="border-white/30 text-xs"
+                    >
+                      Get help
+                    </Button>
+                  </ExternalLink>
+                </div>
```

Comment:
We agreed to remove this button for Settled txs (except for expired txs) and only keep the button for Pending txs.

Haven't been updated on this branch.
<img width="529" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/22cceb0d-42ec-4e48-bd31-9e3a94cce9ea">